client/src/views/Settings/Tabs/EditPrivacy/index.vue

```diff
@@ -80,6 +80,8 @@
 </template>
 
 <script>
+// import { setNotifSettings } from '@/services/notification.js'
```

Comment:
Why commented?

Reply:
It should be in other pr. I pushed unfinished code here by mistake